tests/phpunit/CRM/Dedupe/MergerTest.php

```diff
@@ -1046,6 +1046,42 @@ public function testMigrationOfContactReferenceCustomField() {
     $this->callAPISuccess('CustomGroup', 'delete', ['id' => $activityGroup['id']]);
   }
 
+  /**
+   * Verifies that when two contacts with view only custom fields are merged,
+   * the view only field of the record being deleted is not merged, it is
+   * simply deleted (it should also not be visible on the page).
+   */
+  public function testMigrationOfViewOnlyCustomData() {
+    // Create Custom Fields
+    $createGroup = $this->setupCustomGroupForIndividual();
+    $customField = $this->setupCustomField('TestField', $createGroup);
+
+    // Contacts setup
+    $this->setupMatchData();
+    $originalContactID = $this->contacts[0]['id'];
+    $duplicateContactID = $this->contacts[1]['id'];
+
+    // Update the text custom fields for duplicate contact
+    $this->callAPISuccess('Contact', 'create', [
+      'id' => $duplicateContactID,
+      "custom_{$customField['id']}" => 'abc',
+    ]);
+    $this->assertCustomFieldValue($duplicateContactID, '', "custom_{$customField['id']}");
```

Comment:
It's hard to tell from the line numbers (needs rebase against latest master?) but I think the crm test is failing here because it just set it to abc above, and now is checking that it's blank.